src/groovy/org/transmartproject/export/GENExporter.groovy

```diff
@@ -89,13 +92,13 @@ class GENExporter implements HighDimTabularResultExporter {
                 out << ' '
                 out << row.a2
                 
-                for (/*SnpLzCell*/ Object cell: row) { 
+                for (double[] cell: row) {
                     out << ' '
-                    out << cell.probabilityA1A1
+                    out << cell[0] // probabilityA1A1
                     out << ' '
-                    out << cell.probabilityA1A2
+                    out << cell[1] // probabilityA1A2
                     out << ' '
-                    out << cell.probabilityA2A2
+                    out << cell[2] // probabilityA2A2
```

Comment:
How is the contract for a `MultiValueProjection` relevant for a `non-MultiValueProjection` like "probabilities"? Plus, if the serialization format of the rest-api only supports maps or scalars, then it should be just a matter of wrapping all non-maps non-scalars from `non-MultiValueProjection`s in a map, right? Like `ImmutableMap.of('value', original)`